src/Microsoft.ServiceHub.Framework/ServiceJsonRpcDescriptor.cs

```diff
@@ -139,7 +139,7 @@ public enum MessageDelimiters
 	/// as the type argument to <see cref="JsonRpcConnection.ConstructRpcClient{T}"/> or
 	/// <see cref="IServiceBroker.GetProxyAsync{T}(ServiceRpcDescriptor, ServiceActivationOptions, CancellationToken)"/>.
 	/// </summary>
-	public ImmutableArray<Type> AdditionalClientInterfaces { get; private set; } = ImmutableArray<Type>.Empty;
+	public ImmutableArray<Type> AdditionalServiceInterfaces { get; private set; } = ImmutableArray<Type>.Empty;
```

Comment:
Would this not be considered a breaking change?

Reply:
It would be, except I only introduced that member this week. So no one uses it yet.

Reply:
Fair :)